dog-lover/src/components/DogBreedImages/DogBreedImages.test.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  fireEvent,
+  render,
+  screen,
+  waitForElementToBeRemoved,
+  within,
+} from '../../test-utils';
+import { setupServer } from 'msw/node';
+import { DefaultRequestBody, rest } from 'msw';
+import { DogBreedImages } from './';
+import { MemoryRouter } from 'react-router-dom';
+
+const server = setupServer(
+  rest.get<DefaultRequestBody>(
+    'https://dog.ceo/api/breed/labrador/images',
+    (_req, res, ctx) => {
+      return res(
+        ctx.delay(100),
```

Comment:
Can you explain why you used delay here ?

Reply:
Hello Kosta,

Thank you so much for taking the time to give me your feedback. It is much appreciated!

In response to your first question, I've only used it to make it a bit more realistic. It is definitely not required!